okapi-core/src/test/java/org/folio/okapi/ModuleTest.java

```diff
@@ -1420,8 +1460,20 @@ public void testSystemInterfaces(TestContext context) {
       .then()
       .statusCode(201)
       .log().ifValidationFails()
-      .header("X-Tenant-Perms-Result", expPerms)
       .extract().header("Location");
+
+    body = given()
+        .header("X-Okapi-Tenant", okapiTenant)
+        .get("/permResult")
+        .then()
+        .statusCode(200)
+        .log().ifValidationFails()
+        .extract().body().asString();
+
+    ar = new JsonArray(body);
+    context.assertEquals(2, ar.size());
+    context.assertEquals(new JsonObject(expPerms).encode(), ar.getJsonObject(1).encode());
```

Comment:
encode() does not guarantee the order of the keys. You probably want to compare without encode().

Reply:
yes